crates/daphne-server/examples/service.rs

```diff
@@ -120,11 +121,15 @@ async fn main() -> Result<(), Box<dyn std::error::Error + Sync + Send>> {
         .init();
 
     // hand the router to axum for it to run
-    let serve = axum::Server::bind(&std::net::SocketAddr::new(
-        "0.0.0.0".parse().unwrap(),
-        config.port,
-    ))
-    .serve(router.into_make_service());
+    let serve = axum::serve(
+        TcpListener::bind(std::net::SocketAddr::new(
+            "0.0.0.0".parse().unwrap(),
```

Comment:
could we remove the parsing as not really needed
```suggestion
            IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)),
```

Reply:
I prefer the parsing one because it's less verbose. I think it's more fitting for a code example

Reply:
parsing has been shown to be a source of [bugs](https://blog.cloudflare.com/incident-report-on-memory-leak-caused-by-cloudflare-parser-bug/). 
I would do parsing, unless there is no more secure option.